src/main/java/com/crowdin/client/bundles/model/FileInfoCollectionResourceResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.crowdin.client.bundles.model;
+
+import lombok.Data;
+
+@Data
+public class FileInfoCollectionResourceResponse {
```

Comment:
Let's use the `com.crowdin.client.sourcefiles.model.FileInfo` class instead of duplicating it